### PR TITLE
Social share

### PR DESCRIPTION
**Pull request by nisaChampagne**
# Description

FOR QA AND REVIEWERS: FOLLOW THESE STEPS: https://www.notion.so/QA-and-Staging-Process-for-reviewing-a-pull-request-e751c216650249ffae7d3530f6bfebc2

Implemented social sharing via react modal and react share. This will need a little more work to share the badges but so far we can share the connect our kids url via facebook, linkedin, twitter, and via email

Also thinned out the doughnut chart on public and team dashboards 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change status
- [x] Complete, tested, ready to review and merge 

# How Has This Been Tested?
-Test's are not needed now, but you should have made sure your code was working in some way. 
---eg used postman to make sure endpoints are working

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A: can share connect our kids link via facebook, email, linkedin, and twitter

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts
- [ ] Any env variables or config options that are needed have been stated in the README
- [ ] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [ ] NO API KEYS ARE PUSHED UP

### CODE REVIEW

**Comment by nisaChampagne** [comment]
the first icon on the first badge is not working and thus messed up my share icon. Once that image is fixed, everything will be in its proper placement

**Comment by nisaChampagne** [comment]
Also working on placement of modal when each share icon is clicked

File: src/index.js [hunk 1]
```diff
@@ -34,10 +34,11 @@ ReactDOM.render(
     <Auth0Provider
       domain={config.domain}
       client_id={config.clientId}
-      redirect_uri={window.location.origin+'/dev-sean/'}
+      redirect_uri={window.location.origin}
       onRedirectCallback={onRedirectCallback}
+      returnTo={window.location.origin}
```
Comment (seanwu20) [inline src/index.js]: Does this need to be here?

Comment (nisaChampagne) [reply src/index.js]: It was from our group conversation yesterday of the code snip I asked for, I can remove it

**Comment by seanwu20** [comment]
Looks good, ready for qa

**Pull request merged**